package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "webfonts-loader",
-  "version": "4.2.1",
+  "version": "4.3.1",
```

```suggestion
  "version": "4.2.1",
```

Please do not update the versioning, I'll take care of that myself when releasing.

---

Fixed, sorry!